### PR TITLE
Update loki.write.md

### PR DESCRIPTION
**Pull request by v-zhuravlev**
Docs update: Scheme and path are required in loki.write

### CODE REVIEW

File: docs/sources/flow/reference/components/loki.write.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ local Loki instance:
 ```river
 loki.write "local" {
     endpoint {
-        url = "loki:3100"
+        url = "http://loki:3100/loki/api/v1/push"
```
Comment (clayton-cornell) [inline docs/sources/flow/reference/components/loki.write.md:154]: Just a question... is this only `http` or should it be `https`?

**Comment by v-zhuravlev** [comment]
It can be both , for simple case I would keep it http

Ср, 19 июля 2023 г. в 23:28, Clayton Cornell ***@***.***>:

**Comment by clayton-cornell** [comment]
@grafana/grafana-agent-maintainers ... can someone from the team double check the URL field and rubber stamp this. Then I can get it merged and into the docs.

File: docs/sources/flow/reference/components/loki.write.md (hunk 1 above)
Comment (jcreixell) [reply docs/sources/flow/reference/components/loki.write.md:154]: for local development, it is probably ok to use http (most people won't have certificates configured), for production it should be https, not sure if it's worth a comment

**Review by jcreixell — APPROVED** [review]

**Review by clayton-cornell — APPROVED** [review]
lgtm

**Pull request merged**